scripts/jenkins/qa_openstack.sh

```diff
@@ -446,18 +446,45 @@ if [ -e /etc/tempest/tempest.conf ]; then
     fi
 
     pushd /var/lib/openstack-tempest-test/
+
+    # Handle OpenStack release specific blacklisting of known to fail tests
+    blacklist_args=""
+    case "${cloudsource}" in
+    (openstackrocky)
+        # TODO(fmccarthy): Remove once we have addressed issues causing
+        # failures for the neutron_tempest_plugin tests (SCRD-8681)
+        # If neutron_tempest_plugin is installed then blacklist known
+        # to fail tests
+        if rpm -q python-neutron-tempest-plugin > /dev/null 2>&1; then
+            _blacklist_file=$(mktemp /tmp/neutron_tempest_plugin_blacklist.XXXXXXXX)
+            cat - | tee ${_blacklist_file} << __EOF__
```

Comment:
Yeah the "cat -" is no longer required because I'm doing the tee, my bad; I originally was redirecting to the file but that meant that the output from the script didn't report the blacklist details, so I changed it to a tee...